docs/MO_DG/prepare_model/convert_model/Convert_Model_From_PyTorch.md

```diff
@@ -0,0 +1,53 @@
+5. [Integrate](../../../IE_DG/Samples_Overview.md) the Inference Engine in your application to deploy the model in the target environment.
+
+## Supported Topologies
+
+Here is the list of models that were tested and are guaranteed to be supported.
+It is not a full list of models that can be converted to ONNX\* and to IR.
+
+|Package Name|Supported Models|
+|:----|:----|
+| [Torchvision Models](https://pytorch.org/docs/stable/torchvision/index.html) | alexnet, densenet121, densenet161, densenet169, densenet201, resnet101, resnet152, resnet18, resnet34, resnet50, vgg11, vgg13, vgg16, vgg19 |
+| [Pretrained Models](https://github.com/Cadene/pretrained-models.pytorch) | alexnet, fbresnet152, resnet101, resnet152, resnet18, resnet34, resnet152, resnet18, resnet34, resnet50, resnext101_32x4d, resnext101_64x4d, vgg11 |
+
+**Other supported topologies**
+
+* [ESPNet Models](https://github.com/sacmehta/ESPNet/tree/master/pretrained)
+* [MobileNetV3](https://github.com/d-li14/mobilenetv3.pytorch)
+* F3Net topology can be converted using [instruction](pytorch_specific/Convert_F3Net.md)
+* QuartzNet topologies from [NeMo project](https://github.com/NVIDIA/NeMo) can be converted using [instruction](pytorch_specific/Convert_QuartzNet.md)
+* YOLACT topology can be converted using [instruction](pytorch_specific/Convert_YOLACT.md)
+
+## Export PyTorch\* Model to ONNX\* Format <a name="export-to-onnx"></a>
+
+PyTorch models are defined in a Python\* code, to export such models use `torch.onnx.export()` method.
+Only the basics will be covered here, the step to export to ONNX\* is crucial but it is covered by PyTorch\* framework.
+For more information, please refer to [PyTorch\* documentation](https://pytorch.org/docs/stable/onnx.html).
+
+To export a PyTorch\* model you basically need to obtain the model as an instance of `torch.nn.Module` class and call the export function.
+```python
+import torch
+
+# Instantiate your model. This is just a regular PyTorch model that will be exported in the following steps.
+model = SomeModel()
+# Evaluate the model to switch some operations from training mode to inference.
+model.eval()
+# Create dummy input for the model. It will be used to run the model inside export function. 
+dummy_input = torch.randn(1, 3, 224, 224)
+# Call the export function
+torch.onnx.export(model, (dummy_input, ), 'model.onnx')
+```
+
+## Known Issues
+
+* Not all PyTorch\* operations can be exported to ONNX\* opset 9 which is used by default, as of version 1.8.1.
+It is recommended to export models to opset 11 when export to default opset 9 is not working. In that case, use `opset_version`
+option of the `torch.onnx.export`. For more information about ONNX* opset, refer to the [Operator Schemas](https://github.com/onnx/onnx/blob/master/docs/Operators.md).
```

Comment:
May be it is necessary to mention an example with operation Resize which is correctly converted to ONNX only starting from opset 11.

Reply:
I don't think it is needed. PyTorch will print a warning that opset 11 is recommended for Resize operations.